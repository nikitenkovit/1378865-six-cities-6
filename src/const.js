export const OfferType = {
  APARTMENT: `Apartment`,
  ROOM: `Private Room`,
  HOUSE: `House`,
  HOTEL: `Hotel`
};

export const PathValue = {
  MAIN_SCREEN: `/`,
  FAVORITES_SCREEN: `/favorites`,
  ROOM_SCREEN: `/offer/:id`
};

export const CityCoordinate = {
  AMSTERDAM: {
    latitude: 52.38333,
    longitude: 4.9,
    zoom: 12
  }
}; // скорее всего временно

export const OffersListClassName = {
  CITY_PLACES: `cities__places-list`,
  NEAR_PLACES: `near-places__list`
};

export const BookmarkButtonProperty = {
  ROOM_SCREEN: {
    buttonClass: `property__bookmark-button`,
    iconClass: `property__bookmark-icon`,
    iconWidth: `31`,
    iconHeight: `33`
  },
  OFFER_CARD: {
    buttonClass: `place-card__bookmark-button`,
    iconClass: `place-card__bookmark-icon`,
    iconWidth: `18`,
    iconHeight: `19`
  }
};
