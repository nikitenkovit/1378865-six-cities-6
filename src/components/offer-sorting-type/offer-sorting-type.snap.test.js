import React from "react";
import {render} from "@testing-library/react";
import OfferSortingType from "./offer-sorting-type";
import {DEFAULT_SORTING_TYPE} from "../../const";

describe(`OfferSortingType tests`, () => {
  it(`Should OfferSortingType not active render correctly`, () => {
    const {container} = render(
        <OfferSortingType
          type={DEFAULT_SORTING_TYPE}
          onSelectType={jest.fn}
          isActive={false}
        />);
    expect(container).toMatchSnapshot();
  });

  it(`Should OfferSortingType active render correctly`, () => {
    const {container} = render(
        <OfferSortingType
          type={DEFAULT_SORTING_TYPE}
          onSelectType={jest.fn}
          isActive={true}
        />);
    expect(container).toMatchSnapshot();
  });
});
