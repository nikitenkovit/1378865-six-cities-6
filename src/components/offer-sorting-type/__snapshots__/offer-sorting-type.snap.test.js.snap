// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferSortingType tests Should OfferSortingType active render correctly 1`] = `
<div>
  <li
    class="places__option \\" tabIndex=\\"0\\"
    places__option--active"
  >
    Popular
  </li>
</div>
`;

exports[`OfferSortingType tests Should OfferSortingType not active render correctly 1`] = `
<div>
  <li
    class="places__option \\" tabIndex=\\"0\\"
    null"
  >
    Popular
  </li>
</div>
`;
